diff --git a/node_modules/@aws-sdk/xml-builder/dist-cjs/xml-parser.js b/node_modules/@aws-sdk/xml-builder/dist-cjs/xml-parser.js
index 31499ae..d21356d 100644
--- a/node_modules/@aws-sdk/xml-builder/dist-cjs/xml-parser.js
+++ b/node_modules/@aws-sdk/xml-builder/dist-cjs/xml-parser.js
@@ -16,8 +16,6 @@ const parser = new fast_xml_parser_1.XMLParser({
     tagValueProcessor: (_, val) => (val.trim() === "" && val.includes("\n") ? "" : undefined),
     maxNestedTags: Infinity,
 });
-parser.addEntity("#xD", "\r");
-parser.addEntity("#10", "\n");
 function parseXML(xmlString) {
     return parser.parse(xmlString, true);
 }
